python/lsst/summit/utils/bestEffort.py

```diff
@@ -115,7 +115,7 @@ def __init__(
         self.defaultExtraIsrOptions = defaultExtraIsrOptions
 
         self._cache: dict = {}
-        self._cacheIsForDetector = None
+        self._cacheIsForDetector: str | None = None
```

Comment:
detector is an `int` not a `str`.

Reply:
Tried to find the variable type and finally decided to put as str...

Reply:
You can always ask - always better than guessing 🙂

Reply:
OK, now need an assert or a typing: ignore to remove mypy error